core/arch/arm/plat-imx/conf.mk

```diff
@@ -78,4 +78,8 @@ $(call force,CFG_SECURE_TIME_SOURCE_REE,y)
 CFG_BOOT_SECONDARY_REQUEST ?= y
 endif
 
+ifeq ($(filter y, $(CFG_PSCI_ARM32)), y)
+CFG_CORE_RWDATA_NOEXEC ?= n
+endif
```

Comment:
I recommend `CFG_HWSUPP_MEM_PERM_WXN=n` and keep `CFG_CORE_RWDATA_NOEXEC=y`.
The later insures optee_os mapping splits executable and data, and the former does not enforce write-implies-noexec from SCTLR.

Reply:
ok. I'll use CFG_HWSUPP_MEM_PERM_WXN ?= n in next version.